code/modules/hydroponics/grown/replicapod.dm

```diff
@@ -161,3 +161,95 @@
 
 	parent.update_tray()
 	return result
+
+/obj/item/seeds/replicapod/grodpod
+	name = "Grodpod Seed"
+	desc = "You shouldn't see this"
+	species = "grodpod"
+	plantname = "Grod Cocoon"
+	maturation = 15
+	growthstages = 2
+	var/crown
+
+/obj/item/seeds/replicapod/grodpod/harvest(mob/user)
+	var/obj/machinery/hydroponics/parent = loc
+	var/make_podman = FALSE
+	var/ckey_holder = null
+	var/list/result = list()
+	if(CONFIG_GET(flag/revival_pod_plants))
+		if(ckey)
+			for(var/mob/M in GLOB.player_list)
+				if(isobserver(M))
+					var/mob/dead/observer/O = M
+					if(O.ckey == ckey && O.can_reenter_corpse)
+						make_podman = TRUE
+						break
+				else
+					if(M.ckey == ckey && M.stat == DEAD && !M.suiciding)
+						make_podman = TRUE
+						break
+		else //If the player has ghosted from his corpse before blood was drawn, his ckey is no longer attached to the mob, so we need to match up the cloned player through the mind key
+			for(var/mob/M in GLOB.player_list)
+				if(mind && M.mind && ckey(M.mind.key) == ckey(mind.key) && M.ckey && M.client && M.stat == DEAD && !M.suiciding)
```

Comment:
this is such a weird check, are you sure there isn't already grab player by ckey proc in the code?